hibernate-core/src/main/java/org/hibernate/internal/AbstractSharedSessionContract.java

```diff
@@ -248,6 +248,14 @@ protected final void setUpMultitenancy(SessionFactoryImplementor factory, LoadQu
 		}
 	}
 
+	protected final void setupAutoEnabledFilters(SessionFactoryImplementor factory, LoadQueryInfluencers loadQueryInfluencers) {
```

Comment:
I assume that this method should have been deleted from the pull request, since it currently has no callers.

@beikov @gregoriopalama